managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -1095,20 +1095,19 @@ public void deleteCursorFailed(ManagedLedgerException exception, Object ctx) {
     }
 
     @Override
-    public ManagedCursor newNonDurableCursor(Position startCursorPosition) throws ManagedLedgerException {
+    public ManagedCursor newNonDurableCursor(Position mdPosition) throws ManagedLedgerException {
```

Comment:
Why need to change this name?

Reply:
> Why need to change this name?

Users may assume it is `readPosition` or `markDeletePosition`, it is not clear.

Reply:
Reverted, it should be modified by a separate PR